app/renderer/network.html

```diff
@@ -1,26 +1,34 @@
 <!DOCTYPE html>
 <html lang="en" class="responsive desktop">
-    <head>
-        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
-        <meta name="viewport" content="width=device-width">
-        <title>Zulip - Network Troubleshooting</title>
-        <link rel="stylesheet" href="css/network.css" type="text/css" media="screen">
-    </head>
-    <body>
-        <div id="content">
-            <div id="picture"><img src="img/zulip_network.png"></div>
-            <div id="title">We can't connect to this organization</div>
-            <div id="subtitle">This could be because</div>
-            <ul id="description">
-                <li>You're not online or your proxy is misconfigured.</li>
-                <li>There is no Zulip organization hosted at this URL.</li>
-                <li>This Zulip organization is temporarily unavailable.</li>
-                <li>This Zulip organization has been moved or deleted.</li>
-            </ul>
-            <div id="buttons">
-                <div id="reconnect" class="button">Reconnect</div>
-                <div id="settings" class="button">Settings</div>
-            </div>
-        </div>
-    </body>
+	<head>
+   		<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    	<meta name="viewport" content="width=device-width">
+    	<title>Zulip - Network Troubleshooting</title>
+    	<link rel="stylesheet" href="css/network.css" type="text/css" media="screen">
+	</head>
+	<body>
+    	<div class="container">
+        	<div class="maingraphic">
+            	<img src="img/zulip_network.png">
+        	</div>
+        	<div id="content">
+            	<div id="title">We can't connect to this organization</div>
+            	<div id="subtitle">
+                	<ul>
+                    	<li>This could be because</li>
+	                </ul>
+    	        </div>
+        	    <ul id="description">
+            	    <li>You're not online or your proxy is misconfigured.</li>
+                	<li>There is no Zulip organization hosted at this URL.</li>
+	                <li>This Zulip organization is temporarily unavailable.</li>
+    	            <li>This Zulip organization has been moved or deleted.</li>
+	            </ul>
+    	        <div id="buttons">
+        	        <div id="reconnect" class="button">Reconnect</div>
+            	    <div id="settings" class="button">Settings</div>
```

Comment:
@samyakgaur do you understand what's going on here?

Reply:
@akashnimare Yeah we will have to link it to the respective page. Or should i remove it ?

Reply:
Ideally, we should be using button tag here -
```
<div class="button-container">
<button id="reconnect">Reconnect</button>
<button id="settings">Settings</button>
</div>
```